consignment-service/proto/consignment.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package go.micro.srv.consignment;s
```

Comment:
Maybe change package name to main
Also trailing 's' not needed

Reply:
Ok